content/asset-tracking/using-the-sdks.textile

```diff
@@ -0,0 +1,232 @@
+---
+title: Using the SDKs
+section: asset-tracking
+index: 20
+meta_description: "Ably Asset Tracking SDKs provide an easy way to track multiple assets with realtime location updates powered by the Ably realtime network and Mapbox Navigation SDK with location enhancement."
+meta_keywords: "Ably, Asset Tracking SDK, Mapbox, Mapbox Navigation SDK, Ably realtime"
+languages:
+  - javascript
+  - android
+  - objc
+  - java
+  - swift
+jump_to:
+  Help with:
+    - Supported platforms#supported-platforms
+    - Repositories#repositories
+    - Prerequisites#prerequisites
+    - Installing the SDK#installing-sdk
+    - Using the Publisher SDK#publisher-sdk
+    - Using the Subscriber SDK#subscriber-sdk
+    - See also#see-also
+---
+
+This topic explains how to get started using the Ably Asset Tracking SDKs.
+
+h2(#supported-platforms). Supported platforms
+
+There are two Asset Tracking SDKs, one for publishing and one for subscribing. The following platforms are supported:
+
+* Android (Java and Kotlin) - publisher and subscriber SDK
+* iOS (Objective-C and Swift) - publisher and subscriber SDK
+* Web (JavaScript) - subscriber SDK
+
+h2(#repositories). SDK repositories
+
+The SDKs can be found in the following GitHub repositories:
+
+* "Android":https://github.com/ably/ably-asset-tracking-android
+* "iOS":https://github.com/ably/ably-asset-tracking-cocoa
+* "JavaScript":https://github.com/ably/ably-asset-tracking-js
+
+h2(#prerequisites). Prerequisites
+
+You need to have a suitable developer environment installed, for example:
+
+* Android - "Android Studio":https://developer.android.com/studio
+* iOS - "Xcode":https://developer.apple.com/xcode/
+* JavaScript - any suitable environment of your choice 
+
+You also need to have suitable credentials for the various SDK components:
+
+* @ABLY_API_KEY@ - Your Ably API key
+* @MAPBOX_DOWNLOADS_TOKEN@ - Mapbox credentials
+* @MAPBOX_ACCESS_TOKEN@ - Mapbox credentials
+* @GOOGLE_MAPS_API_KEY@ - Google Maps credentials
+
+You can set these values in your @~/.gradle/gradle.properties@ file.
```

Comment:
This is specific to Android, this line is not applicable to iOS and Javascript

Reply:
Clarified.